tool/teleport/common/teleport_test.go

```diff
@@ -111,6 +116,22 @@ func TestTeleportMain(t *testing.T) {
 		require.Equal(t, "10.5.5.5", conf.AdvertiseIP)
 		require.Equal(t, map[string]string{"a": "a1", "b": "b1"}, conf.SSH.Labels)
 	})
+
+	t.Run("Bootstrap", func(t *testing.T) {
+		_, cmd, conf := Run(Options{
+			Args: []string{
+				"start", "--roles=node",
+				"--labels=a=a1,b=b1",
+				"--config=" + configFile,
+				"--bootstrap=" + bootstrapFile,
+			},
+			InitOnly: true,
+		})
+		require.Equal(t, "start", cmd)
+		require.Equal(t, 2, len(conf.Auth.Resources))
+		require.Equal(t, "github", conf.Auth.Resources[0].GetKind())
+		require.Equal(t, "role", conf.Auth.Resources[1].GetKind())
```

Comment:
Should we do a deeper assertion at the resources or is this enough? (If yes, please leave a comment stating why.)